plugin/src/main/java/io/snyk/eclipse/plugin/views/snyktoolview/SnykToolView.java

```diff
@@ -96,9 +99,21 @@ public void selectionChanged(SelectionChangedEvent event) {
 						LSPEclipseUtils.open(fileNode.getPath().toUri().toASCIIString(),
 								issueTreeNode.getIssue().getLSP4JRange());
 					}
+					boolean deltaEnabled = Preferences.getInstance()
+							.getBooleanPref(Preferences.FILTER_DELTA_NEW_ISSUES);
+					if (node instanceof ContentRootNode && deltaEnabled) {
+						ContentRootNode contentNode = (ContentRootNode) node;
+						String projectPath = contentNode.getPath().toString();
+						String[] localBranches = folderConfigs.getLocalBranches(projectPath).toArray(new String[0]);
```

Comment:
here we would use the signature change above, passing the `Path` object to the folderConfigs